spec/mailers/previews/customer_mailer_preview.rb

```diff
@@ -55,6 +55,12 @@ def recovered_from_link
     CustomerMailer.recovered_from_link(recovered_record)
   end
 
+  def held_bike_email
+    target = Bike.stolen.limit(1).first
+    match = Bike.held.limit(1).first
+    CustomerMailer.held_bike_email(target, match)
+  end
```

Comment:
If all the data is in the CustomerMailer, you don't need to build something here.

This can just be `CustomerContact.held_bike_notification.last`. The other existing email previews should be similarly scoped, since there are no longer just one type of CustomerContact.

Reply:
Corollary of this (and other comments on this review) - I'm not certain `externally_held_bike_notification` needs to be a separate kind of `CustomerContact`. I think having scopes for both types makes sense, so it might make sense to have different kinds, but I think having them route through the same mailer method, and just render different HTML would be nice

Reply:
> I think having scopes for both types makes sense, so it might make sense to have different kinds, but I think having them route through the same mailer method, and just render different HTML would be nice

That's the plan at the moment — they have different `kind` enums but use the same mailer. But I can collapse the `kinds` if you think it's overly granular.

Reply:
> If all the data is in the CustomerMailer, you don't need to build something here.

> This can just be CustomerContact.held_bike_notification.last. The other existing email previews should be similarly scoped, since there are no longer just one type of CustomerContact.

Can you internalize your pronoun-antecedents here?